content/css/jenkins.css

```diff
@@ -920,12 +921,13 @@ blockquote {
 
 .toc {
```

Comment:
Needs this added somewhere so that scroll bars are dark:

```
html {
  color-scheme: light dark;
}
```

(see https://dev.to/madsstoumann/dark-mode-in-3-lines-of-css-and-other-adventures-1ljj)

Reply:
Good spot - thanks!